src/Paramore.Brighter.DynamoDb/GUIDConverter.cs

```diff
@@ -8,10 +8,9 @@ public class GUIDConverter : IPropertyConverter
     {
         public DynamoDBEntry ToEntry(object value)
         {
-            var uuid = (Guid)value;
-            if (uuid == null)
```

Comment:
Hard cast will throw and `uuid` will never be `null`. Changing for a version that throws the intended exception